src/frontend/src/views/GroupMembersPage/components/MembersTab/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Tab } from "czifui";
+import React, { useState } from "react";
+import { Table } from "src/common/components/library/Table";
+import { TabEventHandler } from "../../index";
+import { Header, StyledTabs } from "./style";
+
+type TabType = "active" | "invitations";
+
+interface Props {
+  invites: any[];
+  members: any[];
+}
+
+const MembersTab = ({ invites, members }: Props): JSX.Element => {
+  const [tabValue, setTabValue] = useState<TabType>("active");
```

Comment:
nit: Feels like TabType should be an enum

Reply:
It's in the follow up pr